mythril/src/virtdev/mod.rs

```diff
@@ -497,8 +516,22 @@ mod test {
         }
     }
 
+    impl VirtualMachineConfig {
+        // Default config used for testing
+        pub fn default() -> VirtualMachineConfig {
```

Comment:
It doesn't look like this is used outside of this module, so rather than defining this `default`, I'd rather have a method in the `mod test` below that just returns just a `VirtualMachineConfig` and use that in the tests.